wa/workloads/drarm/__init__.py

```diff
@@ -0,0 +1,109 @@
+#    Copyright 2023 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+import os
+import pandas as pd
+from wa import ApkWorkload, Parameter
+from devlib.exception import TargetStableCalledProcessError
+
+
+class DrArm(ApkWorkload):
+
+    name = 'drarm'
+    package_names = ['com.Arm.DrArm']
+    activity = 'com.unity3d.player.UnityPlayerActivity'
+    install_timeout = 200
+    description = """
+    Dr. Arm’s Amazing Adventures is a “Souls-Like” Mobile Action Role Playing Game developed at Arm.
```

Comment:
Is this app publicly available? If so would it be possible to provide a link to where the application can be obtained from?

Reply:
Hm I don't think so, it's an Arm-internal demo. It's been shown publicly like here:

https://community.arm.com/arm-community-blogs/b/graphics-gaming-and-vr-blog/posts/how-to-use-adaptive-performance

But we don't have the actual app published anywhere outside the internal systems as far as I know